src/lib/loadBatchPriceInsights.ts

```diff
@@ -0,0 +1,47 @@
+import gql from "lib/gql"
+import { extractNodes } from "lib/helpers"
+
+const MAX_MARKET_PRICE_INSIGHTS = 50
```

Comment:
**(note):** We should check if we ever fetch more than 50 artworks with insights in Eigen

Reply:
I can see that we get 30 artworks in my collection in the first query, should I change the value here to 30?